openshift/core.app.yaml

```diff
@@ -197,6 +197,8 @@ objects:
     type: ClusterIP
     sessionAffinity: null
 parameters:
+- name: IMAGE
+  value: registry.devshift.net/fabric8-services/fabric8-wit
```

Comment:
@jmelis this should be `prod.registry.devshift.net...` ?

Reply:
@sbose78 this is just the default value that OpenShift template substitutes, if env var `IMAGE` is not set.

Reply:
@sbose78: like @surajssd pointed out this is overriden by: https://github.com/openshiftio/saas-openshiftio/blob/master/dsaas-services/core.yaml#L11

We left the CentOS container as a default so it's easier to test.